### PR TITLE
Fix issue 1 header size for songs

### PR DESCRIPTION
**Pull request by jessesquires**
I don't *think* this was intentional.

Just noticed and it feels a bit weird to be different sizes. 😄

### CODE REVIEW

File: _posts/2020-05-01-issue-1.md [hunk 1]
```diff
@@ -59,33 +59,33 @@ Two songs this week:
...
-_Didn't grasp what I had until_  
+_A sudden fatal tenant_
+_Seizes room_
+_Didn't grasp what I had until_
 _It was robbed absent too soon_
 
 Says the band, “All profits from Bandcamp will proceed directly to Médecins Sans Frontières who right now are doing their best to provide aid where it so urgently is needed the most.”
 
-##### [GmacCash - On Strike (Shot By DaPoundMedia)](https://www.youtube.com/watch?v=uJOHvQ9cL38)
+#### [GmacCash - On Strike (Shot By DaPoundMedia)](https://www.youtube.com/watch?v=uJOHvQ9cL38)
```
Comment (jessesquires) [inline _posts/2020-05-01-issue-1.md:74]: Fix is here. (this was h5 instead of h4)

Sorry, my edit trims trailing spaces automatically. 😬

Comment (anaulin) [reply _posts/2020-05-01-issue-1.md:74]: Pretty sure that was my bug. Thanks for fixing.

(And thanks for trimming the whitespace, too. I usually set my editors for that too, heh.)

**Comment by dellsystem** [comment]
so unfortunately the trailing whitespace is actually necessary!! it's a markdown thing - that's how you force linebreaks (at least for vanilla markdown; some variants behave differently).

without the two spaces at the end of the line, here's how it looks 
image

**Comment by dellsystem** [comment]
im just going to fix the h5 ->h4 thing in master, rather than messing about with this PR (in case i break something), is that ok @jessesquires? thanks for catching this!

**Comment by jessesquires** [comment]
> so unfortunately the trailing whitespace is actually necessary!! it's a markdown thing - that's how you force linebreaks (at least for vanilla markdown; some variants behave differently).

lol wow. TIL. 😄 

> im just going to fix the h5 ->h4 thing in master, rather than messing about with this PR

sounds good! 👍

**Comment by anaulin** [comment]
> > so unfortunately the trailing whitespace is actually necessary!! it's a markdown thing - that's how you force linebreaks (at least for vanilla markdown; some variants behave differently).

> lol wow. TIL. 😄

I also did not know this. I am now very saddened and require more pizza. 🍕

**Comment by jessesquires** [comment]
>  I am now very saddened and require more pizza. 🍕

LOL me too 😄 

Wendy, please deliver us pizza.

**Comment by stationaery** [comment]
Ana does this tshirt help?

**Comment by anaulin** [comment]
I don't see a t-shirt! Show me the t-shirt! Although carbs >> t-shirt, when it comes to sadness remedies...

**Comment by stationaery** [comment]
Hm! Is this
[image: Screen Shot 2020-05-02 at 9.34.26 PM.png]
not coming through?

**Comment by anaulin** [comment]
Sadness intensifies.


I've never tried to add images via e-mail in a GitHub conversation, so I don't know how it works (or if it does at all). I'm adding all my images directly in the web interface (in https://github.com/techworkersco/techworkersco.github.io/pull/42)

**Comment by anaulin** [comment]
...and then my image upload broke. I fixed it, but you can only see the fixed version if you go to GitHub, the email doesn't get resent. 😿

**Pull request closed without merging**